cli.js

```diff
@@ -10,9 +10,11 @@ const help = require('help-me')({
 })
 const start = require('./start')
 const generate = require('./generate')
+const generatePlugin = require('./generate-plugin')
 
 commist.register('start', start.cli)
 commist.register('generate', generate.cli)
+commist.register('generate-plugin', generatePlugin.cli)
```

Comment:
Can you please check that commist [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) keeps picking up `fastify gen` to generate apps, and maybe `fastify plugin` to generate a plugin?

Maybe we could even change the commands and have `fastify generate-application` or `fastify app` to generate and application, and `fastify plugin` to generate a plugin.

Reply:
I'm a bit confused. My goal is to generate a README from an existing plugin. I don't want to implement a plugin generator.

Reply:
ok then, it should be something like "gen-readme"

Reply:
`generate-app` will still work because of Levenshtein distance